cfn-resources/encryption-at-rest/cmd/resource/resource.go

```diff
@@ -50,12 +48,14 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		return *modelValidation, nil
 	}
 
-	// Create MongoDb Atlas Client using keys
-	client, err := util.CreateMongoDBClient(*currentModel.ApiKeys.PublicKey, *currentModel.ApiKeys.PrivateKey)
-	if err != nil {
-		_, _ = logger.Warnf(constants.ErrorCreateMongoClient, err)
-		return progressevents.GetFailedEventByCode(fmt.Sprintf("Failed to Create Client : %s", err.Error()),
-			cloudformation.HandlerErrorCodeInvalidRequest), nil
+	if currentModel.Profile == nil || *currentModel.Profile == "" {
+		currentModel.Profile = aws.String(util.DefaultProfile)
```

Comment:
[q] In others pr's, I see: 

```
if currentModel.Profile == nil {
		currentModel.Profile = pointy.String(util.DefaultProfile)
	}
```
I think we should standardise on an approach - this one looks more complete & removed the need for pointy. WDYK?

Reply:
I agree. We should handle also the scenario `*currentModel.Profile == ""`